digiwf-integrations/digiwf-openai-integration/digiwf-openai-integration-starter/src/main/java/de/muenchen/oss/digiwf/openai/integration/properties/AzureIntegrationProperties.java

```diff
@@ -0,0 +1,28 @@
+package de.muenchen.oss.digiwf.openai.integration.properties;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.annotation.Validated;
+
+@Getter
+@Setter
+@Validated
+@Component
+@ConfigurationProperties(prefix = "de.muenchen.oss.digiwf.openai.service.azure")
```

Comment:
```suggestion
@ConfigurationProperties(prefix = "digiwf.openai.service.azure")
```

Wir wollten das in Zukunft so machen oder? Wäre dann meiner Meinung nach auch sauber gleich zu z.B. `spring.`